client/blocks/reader-author-link/index.jsx

```diff
@@ -2,32 +2,44 @@
  * External dependencies
  */
 import React from 'react';
+import { get, includes } from 'lodash';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
-import { recordAction, recordGaEvent, recordTrackForPost } from 'reader/stats';
+import * as stats from 'reader/stats';
 
-const ReaderAuthorLink = ( { author, post, siteUrl, children } ) => {
+const ReaderAuthorLink = ( { author, post, siteUrl, children, className } ) => {
 	const recordAuthorClick = ( { } ) => {
-		recordAction( 'click_author' );
-		recordGaEvent( 'Clicked Author Link' );
+		stats.recordAction( 'click_author' );
+		stats.recordGaEvent( 'Clicked Author Link' );
 		if ( post ) {
-			recordTrackForPost( 'calypso_reader_author_link_clicked', post );
+			stats.recordTrackForPost( 'calypso_reader_author_link_clicked', post );
 		}
 	};
 
 	if ( ! siteUrl ) {
 		siteUrl = author.URL;
 	}
 
+	const authorName = get( author, 'name' );
+	const authorNameBlacklist = [ 'admin' ];
+
+	// If the author name is blacklisted, don't return anything
+	if ( typeof authorName === 'string' && includes( authorNameBlacklist, author.name.toLowerCase() ) ) {
```

Comment:
this should probably use `authorName`. If you use the default above, you can drop the `typeof` check too